packages/rum/README.md

```diff
@@ -447,9 +447,11 @@ datadogRum.init({
 ```
 
 </details>
-
+   
 The `premiumSampleRate` is a percentage of `sampleRate`. If you set `sampleRate` to 60 and `premiumSampleRate` to 50, 40% of sessions are dropped, 30% of sessions are collected as Browser and 30% of sessions are collected as Browser Premium.
 
+{{< img src="real_user_monitoring/browser/example-initialization-snippet.mp4" alt="Example initialization snippet for a browser application with custom percentages" video="true" width="100%" >}}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏This won't render well in Github. AFAIK, you can't add a video to a GitHub README... maybe add just a link to it?

Reply:
It looks like a lot of documentation edits are about clarifying the sample rate options. Maybe having a dedicated page related to sample rate configuration could be an interesting option, so we don't have github-related limitations.

Reply:
```suggestion
```